packages/rich-text/src/plugins/Hyperlink/components/useHyperlinkCommon.tsx

```diff
@@ -0,0 +1,15 @@
+import { FieldAppSDK } from '@contentful/app-sdk';
+
+import { useContentfulEditor } from '../../../ContentfulEditorProvider';
+import { findNodePath, isChildPath } from '../../../internal/queries';
+import { useSdkContext } from '../../../SdkProvider';
+
+export function useHyperlinkCommon(element) {
+  const editor = useContentfulEditor();
```

Comment:
nit: I'm not sure I see the benefits for `editor` and `sdk` feels like adding a layer of complexity rather than removing. `isLinkFocused` and `pathToElement` make sense

Reply:
Please feel free to ignore though just my opinion

Reply:
Hmm yes not sure, I think we need `editor` to calculate `pathToElement`